hazelcast/src/main/java/com/hazelcast/map/impl/MapContainer.java

```diff
@@ -382,29 +381,6 @@ public ConcurrentMap<Integer, IndexRegistry> getPartitionedIndexRegistry() {
         return partitionedIndexRegistry;
     }
 
-    @SuppressWarnings("squid:S1751")
-    public void consumeIndexConfigs(Consumer<IndexConfig> consumer) {
-        // 1. Add from global-indexes
-        if (globalIndexRegistry != null) {
-            add(consumer, globalIndexRegistry);
-            return;
-        }
-
-        // 2. Add from partitioned-indexes
-        for (IndexRegistry entry : partitionedIndexRegistry.values()) {
-            add(consumer, entry);
-            break;
-        }
-    }
-
-    private void add(Consumer<IndexConfig> consumer, IndexRegistry indexRegistry) {
-        InternalIndex[] indexes = indexRegistry.getIndexes();
-        for (int i = 0; i < indexes.length; i++) {
-            IndexConfig config = indexes[i].getConfig();
-            consumer.accept(config);
-        }
-    }
-
```

Comment:
Based on what I see in my local, this part is not related to the fix, just removing dead code.

Reply:
true, it was dead code.